src/utils/wrap-data.ts

```diff
@@ -41,5 +41,17 @@ export function getWrapData(tokenSymbol: string): ComponentWrapData[] {
       integrationName: IntegrationName.erc4626WrapV2Adapter,
       wrapData: ZERO_BYTES,
     },
+    {
+      integrationName: '',
+      wrapData: ZERO_BYTES,
+    },
+    {
+      integrationName: IntegrationName.erc4626WrapV2Adapter,
+      wrapData: ZERO_BYTES,
+    },
+    {
+      integrationName: IntegrationName.erc4626WrapV2Adapter,
+      wrapData: ZERO_BYTES,
+    },
```

Comment:
Updates wrap data to new composition adding:
* USDC
* https://basescan.org/address/0xf42f5795D9ac7e9D757dB633D693cD548Cfd9169#code
* https://basescan.org/address/0xc1256ae5ff1cf2719d4937adb3bbccab2e00a2ca#code